scipy/linalg/_basic.py

```diff
@@ -592,15 +594,20 @@ def solve_banded(l_and_u, ab, b, overwrite_ab=False, overwrite_b=False,
     array([-2.37288136,  3.93220339, -4.        ,  4.3559322 , -1.3559322 ])
 
     """
+    (nlower, nupper) = l_and_u
+    return _solve_banded(nlower, nupper, ab, b, overwrite_ab=overwrite_ab,
+                         overwrite_b=overwrite_b, check_finite=check_finite)
+
 
+@_apply_over_batch(('nlower', 0), ('nupper', 0), ('ab', 2), ('b', '1|2'))
```

Comment:
Possibly worth cover covering when the core dimension is 0 in the tutorial?

Reply:
I'd prefer not to. Conceptually it's even simpler than when the core dimension is 1 or 2, there aren't many functions that have an input with core dimension zero, so I don't think it's very important. And I'm not really sure where it would fit well. Currently the tutorial switches examples only when introducing a generalization or special case.